include/net/nrf_cloud.h

```diff
@@ -229,8 +229,8 @@ struct nrf_cloud_init_param {
 /**
  * @brief Initialize the module.
  *
- * @warning This API must be called exactly once, and it must return
- * successfully.
+ * @warning  This API must be called prior to using nRF Cloud
```

Comment:
Is there now an extra space between @warning and This?

Reply:
there... was.